src/codec/CommandCodec.ts

```diff
@@ -17,7 +17,7 @@ export const CommandCodec: Codec<PbCommand, Command> = {
         } else if (message.hasCreateandexercise()) {
             return CreateAndExerciseCommandCodec.deserialize(message.getCreateandexercise()!);
         } else {
-            throw new Error('Deserialization error, unable to discriminate value type - this is likely to be a bug');
+            throw new Error('Command deserialization error, unable to discriminate value type - this is likely to be a bug');
```

Comment:
I assume this error is user-facing? One thing I liked about the error messages printed by node.js was that it told me where the bug likely occurred. All that is to say: is `this is likely a bug` referring to a user error in setting up the command, or a bug in the library?

Reply:
Very good point, I'll improve this error message to point out possible causes and solutions.

Reply:
Tracked [here](https://github.com/digital-asset/daml-js/issues/49).

Reply:
Should be taken care of, feel free to chime in and thanks for the feedback. :smiley: